docs/src/modules/components/AppNavDrawer.js

```diff
@@ -188,49 +197,71 @@ function ProductDrawerButton(props) {
+                </Typography>
+              </div>
             </Link>
             <Link
-              href={ROUTES.materialDocs}
+              href={ROUTES.baseDocs}
               // eslint-disable-next-line material-ui/no-hardcoded-labels
             >
-              Material UI <KeyboardArrowRight fontSize="small" />
+              <div>
+                <ProductLabel>MUI Base</ProductLabel>
+                <Typography color="text.secondary" variant="body2">
+                  Unstyled React components and low-level hooks.
+                </Typography>
+              </div>
             </Link>
+
             <Link
               href={ROUTES.systemDocs}
               // eslint-disable-next-line material-ui/no-hardcoded-labels
             >
-              MUI System <KeyboardArrowRight fontSize="small" />
+              <div>
+                <ProductLabel>MUI System</ProductLabel>
+                <Typography color="text.secondary" variant="body2">
+                  CSS utilities for rapidly laying out custom designs.
+                </Typography>
+              </div>
             </Link>
           </LinksWrapper>
         </li>
         <li role="none">
-          <ProductSubMenu
-            role="menuitem"
-            icon={<IconImage name="product-advanced" />}
-            name={
-              <Box
-                component="span"
-                display="inline-flex"
-                alignItems="center"
-                // eslint-disable-next-line material-ui/no-hardcoded-labels
-              >
-                MUI&nbsp;X
-              </Box>
-            }
-            description="Advanced and powerful components for complex use-cases."
-          />
-          <LinksWrapper>
-            <Link
-              href={ROUTES.dataGridSpace}
-              // eslint-disable-next-line material-ui/no-hardcoded-labels
-            >
-              Data Grid <KeyboardArrowRight fontSize="small" />
-            </Link>
-          </LinksWrapper>
+          <Link
+            href={ROUTES.dataGridSpace}
+            sx={{
+              p: 2,
+              '&:hover': {
+                backgroundColor:
+                  theme.palette.mode === 'dark'
+                    ? alpha(theme.palette.primaryDark[700], 0.4)
+                    : theme.palette.grey[50],
+              },
+            }}
+            // eslint-disable-next-line material-ui/no-hardcoded-labels
+          >
+            <ProductSubMenu
+              role="menuitem"
+              icon={<IconImage name="product-advanced" />}
+              name={
+                <Box
+                  component="span"
+                  display="inline-flex"
+                  alignItems="center"
+                  // eslint-disable-next-line material-ui/no-hardcoded-labels
+                >
+                  MUI&nbsp;X
+                </Box>
+              }
+              description="Advanced and powerful components for complex use-cases."
```

Comment:
I don't want to distract from the main changes here, but I believe the correct usage is "use case." (Though I have observed "use-case" throughout the MUI docs.) https://en.wikipedia.org/wiki/Use_case

```suggestion
              description="Advanced and powerful components for complex use cases."
```

Reply:
Thank you! :)